shopelectro/views/catalog.py

```diff
@@ -33,6 +32,18 @@ def get_paginated_page_or_404(objects, per_page, page_number):
         raise http.Http404('Page does not exist')
 
 
+class CategorySortingOption:
+    def __init__(self, index=0):
+        options = settings.CATEGORY_SORTING_OPTIONS[index]
+        self.label = options['label']
+        self.field = options['field']
+        self.direction = options['direction']
+
+    @property
+    def directed_field(self):
```

Comment:
`as_db_exp` may be better

Reply:
@artemiy312 , y, i thought about smth like this. But this approach has several cons:
- it's not db (sql) exp, but query_set's one
- `exp` or `qs` are shortlands, it'll be good to name smth without it
- `name` or `-name` is too simple concept to explain it in query set terms

I'll add pydoc with examples instead